docs/content/testing/testing-systems.md

```diff
@@ -0,0 +1,136 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Testing GDK Systems
+
+We can test systems functionality like this:
+
+- Construct a world
+- Add systems
+- Execute systems
+- Check state
+- Dispose
+
+See Unite Berlin talk: https://youtu.be/BW9qSy6ZB0A?t=9048
+
+See this example test that does not need the `[SetUp]` or `[TearDown]`
```

Comment:
I think this will need to be moved into its own document that should be covered by UTY-290. I added some comments into UTY-290 to clarify some things.

Reply:
For now, removing the "that does not need the `[SetUp]` or `[TearDown]`